src/pages/RepoPage/CoverageOnboarding/GitHubActions/GitHubActions.tsx

```diff
@@ -37,36 +37,18 @@ function GitHubActions() {
   const uploadToken = orgUploadToken ?? data?.repository?.uploadToken ?? ''
   const tokenCopy = orgUploadToken ? 'global' : 'repository'
 
-  return (
-    <div className="flex flex-col gap-6">
-      <Step1 orgUploadToken={orgUploadToken} owner={owner} repo={repo} />
-      <Step2 tokenCopy={tokenCopy} uploadToken={uploadToken} />
-      <Step3 />
-      <FeedbackCTA />
-      <LearnMoreBlurb />
-    </div>
-  )
-}
-
-type Framework = 'Jest' | 'Vitest' | 'Pytest' | 'Go'
-
-interface Step1Props {
-  orgUploadToken: string | null | undefined
-  owner: string
-  repo: string
-}
+  const [framework, setFramework] = useState<Framework>('Jest')
 
-function Step1({ orgUploadToken, owner, repo }: Step1Props) {
   const frameworkInstructions = {
     Jest: {
       install: 'npm install --save-dev jest',
       run: 'npx jest --coverage',
       workflow: `name: Run tests and upload coverage
-
-on: 
+  
+  on: 
   push
```

Comment:
think the spacing here will cause some formatting issues

<img width="730" alt="Screenshot 2024-11-07 at 2 36 34 PM" src="https://github.com/user-attachments/assets/7ddc5b55-6d30-4b54-b281-ed5ebd670152">

Reply:
I think something went wrong with the copy and paste since I was just trying to move this